docs/source/auth_rules.md

```diff
@@ -548,6 +566,18 @@
     <td><sub>EDIT</sub></td>
     <td><sub>The DID used to create the SCHEMA</sub></td>
   </tr>    
+  
+  <tr>
+    <td><sub>SET_CONTEXT</sub></td>
+    <td><sub>ADD</sub></td>
+    <td><sub>N/A</sub></td>
+  </tr>    
+
+  <tr>
+    <td><sub>SET_CONTEXT</sub></td>
+    <td><sub>EDIT</sub></td>
+    <td><sub>A CONTEXT cannot be edited</sub></td>
```

Comment:
This spreadsheet clarifies who is the Owner of the object regardless whether it can be edited (and BTW whether an object can be edited is defined by Auth Rules which can be customized by a Network Administrators; so can-not-edit default rules for Schema and Context can be changed (in theory) to the ones allowing editing). 

So, I believe something like `The DID used to create the CONTEXT` needs to be there.

Reply:
fixed in [ff6f6d7](https://github.com/hyperledger/indy-node/pull/1430/commits/ff6f6d7db1275f4eaadf43cf17127f8c82390936)